app-utils/swww/autobuild/build

```diff
@@ -0,0 +1,19 @@
+abinfo "Generating swww doc ..."
+./doc/gen.sh
+
+abinfo "Building swww"
+cargo build --release
+
+abinfo "Installing swww ..."
+install -Dvm755 target/release/swww -t "${PKGDIR}"/usr/bin/
+install -Dvm755 target/release/swww-daemon -t "${PKGDIR}"/usr/bin/
+
```

Comment:
Why not use ABTYPE=rust?

Reply:
`ABTYPE=rust` will not build swww-daemon.
will switch to `ABTYPE=rust` after the next version is released. https://github.com/LGFae/swww/pull/330